theano/sparse/sandbox/sp2.py

```diff
@@ -132,7 +136,15 @@ def perform(self, node, (n, p, shape, ), (out, )):
         out[0] = csx_matrix(binomial, dtype=self.dtype)
 
     def grad(self, (n, p, shape, ), (gz,)):
-        return None, None, None
+        comment_n = "No gradient exists for the number of samples in class\
+                     Binomial of theano/sparse/sandbox/sp2.py"
+        comment_p = "No gradient exists for the prob of success in class\
+                     Binomial of theano/sparse/sandbox/sp2.py"
+        return [theano.gradient.grad_undefined(op=self, x_pos=0, x=n,
+                                               comment=comment_n),
+                theano.gradient.grad_undefined(op=self, x_pos=1, x=p,
+                                               comment=comment_p),
+                theano.gradient.disconnected_type()]
```

Comment:
Also, even if all were `grad_undefined`, it is still useful information to know that the gradients are Null because they do not make any mathematical sense, rather than because they are not implemented yet.
For instance, the regular random ops in `raw_random.py` return `grad_undefined`.